resources/suite-runner.mjs

```diff
@@ -25,6 +23,30 @@ export class SuiteRunner {
         this.#params = params;
     }
 
+    get frame() {
```

Comment:
Do we really need to expose all these methods now?
It doesn't seem like this is ever called anywhere??
Ditto for other methods in this class.

Reply:
yeah, since the RemoteSuiteRunner extends the SuiteRunner and the private vars are not available in the RemoteSuiteRunner.

I have to use getters to be able to reference the frame like:
`this.frame.contentWindow.postMessage`

Reply:
Oh, I missed that these are getters, not methods.